src/views/GovernanceTaskList/index.tsx

```diff
@@ -36,20 +36,17 @@ const GovernanceTaskList = () => {
     if (systemIntake.id !== '' && systemIntake.businessCaseId !== '') {
       dispatch(fetchBusinessCase(systemIntake.businessCaseId));
     }
-  });
+  }, [dispatch, systemIntake]);
```

Comment:
Using the entire `systemIntake` here is still pretty broad. For hooks, it's better to be specific with the exact data points you want to use, since the hook will run whenever those data points change. For this particular example, it would be best to use `systemIntake.id` and `systemIntake,businessCaseId`, rather than popping those two values out inside the hook.

Reply:
On a similar note (and nit pick to tighten things up), the conditional can probably just be
`if (systemIntake.id && systemIntake.businessCaseId)`.